src/scaffoldmaker/meshtypes/meshtype_3d_stomach1.py

```diff
@@ -0,0 +1,2473 @@
+                'Refine number of elements through wall': 1
+            },
+        }),
+        'Rat 1': ScaffoldPackage(MeshType_3d_ostium1, {
+            'scaffoldSettings': {
+                'Number of vessels': 1,
+                'Number of elements across common': 2,
+                'Number of elements around ostium': 8,
+                'Number of elements along': 2,
+                'Number of elements through wall': 1,  # not implemented for > 1
+                'Unit scale': 1.0,
+                'Outlet': False,
+                'Ostium diameter': 4.0,
+                'Ostium length': 3.5,
+                'Ostium wall thickness': 0.5,
+                'Ostium inter-vessel distance': 0.0,
+                'Ostium inter-vessel height': 0.0,
+                'Use linear through ostium wall': True,
+                'Vessel end length factor': 1.0,
+                'Vessel inner diameter': 1.25,
+                'Vessel wall thickness': 0.5,
+                'Vessel angle 1 degrees': 0.0,
+                'Vessel angle 1 spread degrees': 0.0,
+                'Vessel angle 2 degrees': 0.0,
+                'Use linear through vessel wall': True,
+                'Use cross derivatives': False,
+                'Refine': False,
+                'Refine number of elements around': 4,
+                'Refine number of elements along': 4,
+                'Refine number of elements through wall': 1
+            },
+        }),
+    }
+
+    @staticmethod
+    def getName():
+        return '3D Stomach 1'
+
+    @staticmethod
+    def getParameterSetNames():
+        return [
+            'Default',
+            'Human 1',
+            'Rat 1']
+
+    @classmethod
+    def getDefaultOptions(cls, parameterSetName='Default'):
+        if 'Rat 1' in parameterSetName:
+            centralPathOption = cls.centralPathDefaultScaffoldPackages['Rat 1']
+            ostiumOption = cls.ostiumDefaultScaffoldPackages['Rat 1']       
+        else:
+            centralPathOption = cls.centralPathDefaultScaffoldPackages['Human 1']
+            ostiumOption = cls.ostiumDefaultScaffoldPackages['Human 1']
+
+        options = {
+            'Central path': copy.deepcopy(centralPathOption),
+            'Number of elements around esophagus': 8,
+            'Number of elements around duodenum': 12,
+            'Number of elements between annulus and duodenum': 6,
+            'Number of elements through wall': 1,
+            'Number of radial elements in annulus': 1, 
+            'Wall thickness': 5.0,
+            'Limiting ridge': False,
+            'Gastro-esophagal junction': copy.deepcopy(ostiumOption),
+            'Gastro-esophagal junction position along factor': 0.35,
+            'Annulus derivative factor': 1.0,
+            'Use cross derivatives': False,
+            'Use linear through wall' : False,
+            'Refine': False,
+            'Refine number of elements around': 1,
+            'Refine number of elements along': 1,
```

Comment:
Merge these to ``Refine number of elements surface`` as they must be the same when you have tricky general linear maps.